src/services/subject.js

```diff
@@ -0,0 +1,9 @@
+const Subject = require('~/models/subject')
+
+const subjectService = {
+  getNamesByCategoryId: async (match) => {
+    return await Subject.find(match).select('name').lean().exec()
```

Comment:
You should delete await, If you're using async, you're already working with promises, so there's no need to add await before return